pkg/oci/oci.go

```diff
@@ -252,7 +273,14 @@ func TagGadgetImage(ctx context.Context, srcImage, dstImage string) (*GadgetImag
 		// Error message not that helpful
 		return nil, fmt.Errorf("resolving src: %w", err)
 	}
-	ociStore.Tag(context.TODO(), targetDescriptor, dst.String())
+
+	if err := ociStore.Tag(context.TODO(), targetDescriptor, dst.String()); err != nil {
+		return nil, fmt.Errorf("tagging image: %w", err)
```

Comment:
Does `Tag` fail if it is already tagged by a previous attempt?

Reply:
It appears to be idempotent:

```bash 
$ sudo ig image tag trace_open foo
$ sudo ig image tag trace_open foo
$ sudo ig image tag trace_open foo